runtime/runc_jailer.go

```diff
@@ -460,9 +460,19 @@ func (j *runcJailer) setDefaultConfigValues(cfg *config.Config, socketPath strin
 // Close will cleanup the container that may be left behind if the jailing
 // process was killed via SIGKILL.
 func (j *runcJailer) Close() error {
-	return j.runcClient.Delete(j.ctx, j.vmID, &runc.DeleteOpts{
-		Force: true,
-	})
+	// Even the jailer's associated context is cancelled,
+	// we'd like to do the cleanups below just in case.
+	ctx := context.Background()
```

Comment:
I think we probably still want some sort of deadline here. Can we use a timeout?

Reply:
Okay. How about 5 seconds?

Reply:
That sounds good to me